teachopencadd/talktorials/002_compound_adme/talktorial.ipynb

```diff
@@ -6,11 +6,11 @@
    "source": [
```

Comment:
<ul><li>... to find desired drugs -&gt; to develop effective drugs</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/TeachOpenCADD/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Done.</p>